addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -210,7 +210,7 @@ export class Rtc extends Record {
     timeouts = new Map();
     /** @type {Map<number, number>} timeoutId by sessionId for download pausing delay */
     downloadTimeouts = new Map();
-    iceServers = Record.attr(DEFAULT_ICE_SERVERS, {
+    iceServers = Record.attr(JSON.parse(JSON.stringify(DEFAULT_ICE_SERVERS)), {
```

Comment:
Can't we just remove the default value? The could should make it right regardless.

Reply:
Yes, I would expect to make type inference, but it cannot with `JSON.parse(JSON.stringify())` apparently, so the default value has no value anymore. I'm removing it.

Reply:
I think the script from Matthieu needs `@type` on attr/compute anyway to correctly build the d.ts.